### PR TITLE
HOCS-2755 - MUI Team Management - Re-De/activate a team

### PR DESCRIPTION
**Pull request by agraham19**
* Add functionality to existing patch endpoint to active/deactivate team

### CODE REVIEW

**Review by fewtrees-ho — CHANGES_REQUESTED** [review]
Thanks Andy :-)

Looks good, just minor suggestions made.

.DS_Store needs to be removed from the commit. It's in .gitignore but somehow made it past!

The approval is pending the High change only.

File: src/main/java/uk/gov/digital/ho/hocs/info/api/TeamService.java [hunk 1]
```diff
@@ -187,6 +187,17 @@ public void updateTeamName(UUID teamUUID, String displayName) {
         log.info("Team with UUID {} name updated to {}.", team.getUuid().toString(), displayName, value(EVENT, TEAM_RENAMED));
     }
 
+    @Transactional
+    public void setTeamActiveFlag(UUID teamUUID, boolean active) {
```
[the remark this answers is not in the document]
Comment (agraham19) [reply src/main/java/uk/gov/digital/ho/hocs/info/api/TeamService.java]: I have addressed this.

File: src/test/java/uk/gov/digital/ho/hocs/info/api/TeamResourceTest.java [hunk 2]
```diff
@@ -255,6 +255,23 @@ public void shouldPatchTeam_UpdateUnitWhenNewNameGiven() {
         verifyNoMoreInteractions(teamService);
     }
 
+    @Test
```
[the remark this answers is not in the document]
Comment (agraham19) [reply src/test/java/uk/gov/digital/ho/hocs/info/api/TeamResourceTest.java]: I have addressed this.

File: src/main/java/uk/gov/digital/ho/hocs/info/api/TeamResource.java [hunk 3]
```diff
@@ -57,6 +57,10 @@ public ResponseEntity patchTeam(@PathVariable UUID teamUuid, @RequestBody PatchT
             teamService.moveToNewUnit(teamPatch.getUnitUuid(), teamUuid);
         }
 
+        if (teamActiveFlagHasChanged(team, teamPatch)) {
```
[the remark this answers is not in the document]
Comment (agraham19) [reply src/main/java/uk/gov/digital/ho/hocs/info/api/TeamResource.java]: I have addressed this.

**Review by fewtrees-ho — APPROVED** [review]
Thanks :-)

**Pull request merged**